cms/templates/course-create-rerun.html

```diff
@@ -11,7 +12,7 @@
 
 <%block name="jsextra">
   <script type="text/javascript">
-    var source_course_key = "${source_course_key | escapejs}";
+    var source_course_key = "${source_course_key | js_escaped_string}";
```

Comment:
Every example of `js_escaped_string` I've seen also applies the `n` filter. Should that be done here, too.